HTTP_API.md

```diff
@@ -0,0 +1,318 @@
+
+event: ai_answer_end
+data: {"sections": {"Alert Explanation": ...}}
+```
+
+---
+
+### 3. `/api/workload_health_check` (POST)
+**Description:** Performs a health check on a specified workload (e.g., a Kubernetes deployment).
+
+#### Request Fields
+| Field                   | Required | Default                                    | Type      | Description                                      |
+|-------------------------|----------|--------------------------------------------|-----------|--------------------------------------------------|
+| ask                     | Yes      |                                            | string    | User's question                                  |
+| resource                | Yes      |                                            | object    | Resource details (e.g., name, kind)              |
+| alert_history_since_hours| No       | 24                                         | float     | How many hours back to check alerts              |
+| alert_history           | No       | true                                       | boolean   | Whether to include alert history                 |
+| stored_instrucitons     | No       | true                                       | boolean   | Use stored instructions                          |
+| instructions            | No       | []                                         | list      | Additional instructions                          |
+| include_tool_calls      | No       | false                                      | boolean   | Include tool calls in response                   |
+| include_tool_call_results| No       | false                                      | boolean   | Include tool call results in response            |
+| prompt_template         | No       | "builtin://kubernetes_workload_ask.jinja2" | string    | Prompt template to use                           |
+| model                   | No       |                                            | string    | Model name to use                                |
+
+#### Example
+```bash
+curl -X POST http://<HOLMES-URL>/api/workload_health_check \
+  -H "Content-Type: application/json" \
+  -d '{
+    "ask": "Why is my deployment unhealthy?",
+    "resource": {"name": "my-deployment", "kind": "Deployment"},
+    "alert_history_since_hours": 12
+  }'
+```
+
+#### Example Response
+```json
+{
+  "analysis": "Deployment 'my-deployment' is unhealthy due to repeated CrashLoopBackOff events.",
+  "sections": null,
+  "tool_calls": [
+    {
+      "tool_call_id": "2",
+      "tool_name": "kubectl_get_events",
+      "description": "Fetch recent events",
+      "result": {"events": "..."}
+    }
+  ],
+  "instructions": [...]
+}
+```
+
+---
+
+### 4. `/api/workload_health_chat` (POST)
+**Description:** Conversational interface for discussing the health of a workload. Maintains context across messages.
```

Comment:
Can you remove 'Maintains context across messages.' - to me this implies the api is stateful but actually it's stateless and relies on the conversation_history passed to the chat for this to work.